config/locales/en.yml

```diff
@@ -70,6 +70,7 @@ en:
     split_qty_error: "Quantity to split should be at least 1 and less than %{qty}"
   orders_package:
     already_dispatched: "This has been already dispatched."
+    order_status_error: "You need to complete processing Order first before dispatching."
```

Comment:
Just refining the english a bit "You need to finish processing the order before dispatching."